vllm/compilation/backends.py

```diff
@@ -401,32 +205,28 @@ def __init__(self, post_grad_passes: Sequence[Callable] = ()):
         # streams, it might not be safe to share a global pool.
         # only investigate this when we use multiple streams
         self.graph_pool = global_graph_pool
-        self.post_grad_passes = post_grad_passes
+
+        # Passes to run on the graph post-grad.
+        self.post_grad_pass_manager = PostGradPassManager(
```

Comment:
What passes are expected to be passed in at __init__ time vs __call__ time?

Reply:
This just allows anyone manually constructing `VllmBackend` to add additional passes. Generally known passes will depend on config and should be done at call time. I can add a comment

Reply:
This was removed